lib/fizzy/execute.cpp

```diff
@@ -19,6 +20,11 @@ namespace
 // code_offset + imm_offset + stack_height
 constexpr auto BranchImmediateSize = 3 * sizeof(uint32_t);
 
+constexpr uint32_t F32AbsMask = 0x7fffffff;
```

Comment:
Could go crazy and use `std::numeric_limits<uint32_t>::max() >> 1`, but the constant is nicer.